configure.ac

```diff
@@ -366,3 +366,9 @@ AC_OUTPUT()
 if [ test "`nc-config --version`" == "netCDF 4.7.4" ]; then
   AC_MSG_RESULT([Warning: netCDF v4.7.4 is loaded. Incompatible fms2_io tests will be skipped and there may be netCDF issues with model runs.])
 fi
+
+# Check if gcc is 11.1 for class(*) select type bug
+if [ test -n "`$FC --version | grep GNU | grep 11\..\..`" ]; then
+  AC_MSG_RESULT([Warning: Compilation with gcc and gfortran 11 is unsupported by this version\
```

Comment:
Should this be a warning or a fatal?  I don't think we should allow compiling with gcc11 at all.

Reply:
I had it as an error originally but figured since it still compiles successfully compilation should still be allowed. I'll switch it to an error, odds are the warning will get ignored half the time anyway.